docs/getting-started.md

```diff
@@ -22,10 +22,11 @@ required) list that will help join our projects:
 In our work, we use multiple online platforms, and you may want to ensure you
 registered / have an account at:
 
-- [Microsoft live](https://live.com) (to share documents, access Teams and
-  Azure)
-- [GitHub](https://github.com) (to collaborate on our projects and code)
-- [Docker Hub](https://hub.docker.com/) (to access and publish container images)
```

Comment:
why is the code highlighted in red?

Reply:
cause it is deleted text

Reply:
Yep that's deleted text. Just like the previous one. Takes some time to get used to the difference view, this is a good  find